custom/nic_compliance/utils.py

```diff
@@ -61,8 +61,7 @@ def extract_password(obfuscated_password):
             # completely
             return ''
     else:
-        # return the password received AS-IS
-        return obfuscated_password
+        return False
```

Comment:
I think returning `None` would be better than False

Reply:
makes sense, updated.